src/app/results/results.component.html

```diff
@@ -0,0 +1,43 @@
+<ng-template #resultsBlock>
+    <div *ngFor="let result of results">
+        <app-results-item [result]="result"></app-results-item>
+    </div>
+</ng-template>
+<ng-template #emptyBlock>You have no results yet. Lets add some!</ng-template>
+
+<div>
+    <h2>Your results!</h2>
+    <div *ngIf="!!results.length; then resultsBlock else emptyBlock"></div>
```

Comment:
I like the `ngIfElse` directive to show a placeholder when no data is available!